unlock-app/src/hooks/useLockSettings.ts

```diff
@@ -0,0 +1,47 @@
+import { SubgraphService } from '@unlock-protocol/unlock-js'
+import { useWeb3Service } from '~/utils/withWeb3Service'
+
+interface LockSettingsProps {
+  lockAddress: string
+  network: number
+}
+
+export function useLockSettings() {
+  const web3Service = useWeb3Service()
+  const subgraph = new SubgraphService()
+
+  const getIsRecurringPossible = async ({
+    lockAddress,
+    network,
+  }: LockSettingsProps) => {
+    const lock = await subgraph.lock(
+      {
+        where: {
+          address_in: [lockAddress],
+        },
+      },
+      {
+        network,
+      }
+    )
+    const refund = await await web3Service.refundPenaltyBasisPoints({
+      lockAddress,
+      network,
+    })
+
```

Comment:
This is wrong. The renewals has nothing to do with recurring.

Reply:
Sorry, can you please provide more details on this note?